src/Mechanical/TranslationalPosition/sensors.jl

```diff
@@ -0,0 +1,81 @@
+"""
+    ForceSensor(; name)
+
+Linear 1D force input sensor.
```

Comment:
```suggestion
Linear 1D force sensor, measures the force between two flanges.
```

Reply:
fixed in [35a3947](https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/303/commits/35a3947d4f0b27ee155716336bc681c9ff55d06f)